--- a/Makefile	2018-04-08 22:57:57.866574873 +0800
+++ b/Makefile	2018-04-08 23:01:10.816378216 +0800
@@ -827,7 +827,8 @@
 	@mkdir -p $(dir $@)
 	$(pCC) $(CFLAGS) -I${ROOTDIR}/src -c -o $@ $<
 
-$(BUILDDIR)/bundle.c: check_dvb_scan make_webui
+$(BUILDDIR)/bundle.c: $(DVBSCAN-yes) make_webui
+	find src/webui/static/img/doc/ -delete
 	@mkdir -p $(dir $@)
 	$(pMKBUNDLE) -o $@ -d ${BUILDDIR}/bundle.d $(BUNDLE_FLAGS) $(BUNDLES:%=$(ROOTDIR)/%)
 
